src/bp/ui-studio/src/web/views/OneFlow/diagram/Forms/VariableForm/index.tsx

```diff
@@ -86,6 +86,38 @@ const VariableForm: FC<Props> = ({
   ]
   const selectedVisibility = visibilityOptions.find(({ value }) => value === variableVisibility.current)
 
+  const { advancedSettings, fields, inputType, canAddOptions } = selectedVariableType?.config ?? {}
+
+  const advanced: FormField[] = [...(advancedSettings || [])]
+
+  if (currentFlow.type === 'reusable') {
+    advanced.push(
+      {
+        type: 'text',
+        key: 'label',
+        placeholder: 'optional',
+        label: 'label'
+      },
+      {
+        type: 'text',
+        key: 'placeholder',
+        placeholder: 'optional',
+        label: 'placeholder'
+      }
+    )
+
+    if (canAddOptions) {
+      advanced.push({
+        type: 'text_array',
+        key: 'elements',
+        label: 'Available Elements',
+        group: {
+          minimum: 0
```

Comment:
this shouldn't be required.

Reply:
Yeah, should be none by default, but everywhere else it's minimum 1, so i made this change to avoid breaking something elsewhere